package TryCatch;

import java.io.IOException;

public class Test3 extends Test2 {
	
	public void show1() {
		
			int num = 100/0;
		
	}

}
